package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "quadbin-js",
+  "description": "Utility functions for working with Quadbins",
+  "license": "MIT",
+  "private": true,
+  "keywords": ["quadbin", "webgl", "visualization"],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/CartoDB/quadbin-js.git"
+  },
+  "scripts": {
+    "clean": "ocular-clean",
```

Comment:
I'd include ocular dependency here. As it's not related with vis.gl I'd use something more standard

Reply:
You mean you *wouldn't* include it?